docs/usage/validators.md

```diff
@@ -388,88 +414,6 @@ assert jane_doe.name == 'Jane Doe'
 assert john_doe.name == 'John Doe'
 ```
 
-As it is obvious, repetition has been reduced and the models become again almost
-declarative.
-
-!!! tip
-    If you have a lot of fields that you want to validate, it usually makes sense to
-    define a help function with which you will avoid setting `allow_reuse=True` over and
-    over again.
-
-## Model validators
```

Comment:
Just to verify, these changes just move stuff, yes? Or are there detailed changes?

Reply:
These remove outdated documentation

Reply:
Got it.